zerver/lib/actions.py

```diff
@@ -1415,7 +1415,7 @@ def do_add_subscription(user_profile, stream, no_log=False):
         event = dict(type="subscription", op="peer_add",
                      subscriptions=[stream.name],
                      user_email=user_profile.email)
-        send_event(event, user_ids)
+        send_event(event, active_user_ids(user_profile.realm))
```

Comment:
Cool. There are few more tests that are failing. I will fix those and update this PR. Sounds okay?